lib/loader/fs-resolver.js

```diff
@@ -31,6 +31,16 @@ function getFile(filePath, deferred, secondPath) {
 }
 
 /**
+ * Returns true if `p` is an absolute path.
+ * TODO: Replace with path.isAbsolute() once node 0.10 support is no longer needed.
+ * @param  {string}  p path
+ * @return {boolean}
+ */
+function isAbsolute(p) {
+  return path.normalize(p + '/') === path.normalize(path.resolve(p) + '/');
+}
```

Comment:
Just use the polyfill for now: https://github.com/sindresorhus/path-is-absolute The above will definitely break in some cases.